koiki/delivery.py

```diff
@@ -54,3 +55,20 @@ def send_mail_to_vendor(self):
         send_mail.attach(f"{self.shipment_id}.pdf", pdf.read(), "application/pdf")
 
         send_mail.send(fail_silently=False)
+
+    def send_error_mail_to_admin(self):
+
+        context = {
+            'url_wc_order': f'{wcfmmp_host}area-privada/orders-details/{self.order_id}',
+            'req_body': self.req_body,
+            'error_returned': self.data.get("mensaje", ""),
```

Comment:
I'd be more explicit for customer support to provide `missatge d'error no proporcionat` as default error.